src/openrct2/core/Random.hpp

```diff
@@ -118,7 +118,9 @@ namespace Random
             return std::numeric_limits<result_type>::max();
         }
 
-        explicit RotateEngine(result_type seed_value = default_seed)
+        RotateEngine() = default;
+
+        explicit RotateEngine(result_type seed_value)
```

Comment:
@ZehMatt I had to change this signature as well to make it compile - I’d like to know if this is correct.

Reply:
That's fine, not sure why it had that default parameter anyway.